appveyor.yml

```diff
@@ -1,7 +1,7 @@
 build: off
 
 branches:
-  only: [master, /^features\..*]
+  only: [master, "/^feature\\..*/", "/^\\d+\\.\\d+\\.(\\d+([+-].*)?|x)$/"]
```

Comment:
A comment would be nice explaining the second string.

Reply:
Done.